org.eclipse.jdt.ui/plugin.xml

```diff
@@ -610,10 +610,25 @@
          <showInPart id="org.eclipse.jdt.ui.PackageExplorer"/>
       </perspectiveExtension>
 
-      <perspectiveExtension targetID="org.eclipse.jdt.ui.JavaPerspective">
+      <perspectiveExtension
+            editorOnboardingImage="$nl$/icons/full/onboarding_jperspective.png"
+            editorOnboardingText="perspectiveExtension.editorOnboardingText"
+            targetID="org.eclipse.jdt.ui.JavaPerspective">
          <showInPart id="org.eclipse.jdt.ui.PackageExplorer"/>
          <showInPart id="org.eclipse.team.ui.GenericHistoryView"/>
          <showInPart id="org.eclipse.ui.navigator.ProjectExplorer"/>
+         <editorOnboardingCommand
+               id="org.eclipse.ui.window.quickAccess">
+         </editorOnboardingCommand>
+         <editorOnboardingCommand
+               id="org.eclipse.ui.window.showKeyAssist">
+         </editorOnboardingCommand>
+         <editorOnboardingCommand
+               id="org.eclipse.ui.newWizard">
+         </editorOnboardingCommand>
+         <editorOnboardingCommand
+               id="org.eclipse.jdt.ui.navigate.open.type">
+         </editorOnboardingCommand>         
       </perspectiveExtension>
 
       <perspectiveExtension targetID="org.eclipse.jdt.ui.JavaBrowsingPerspective">
```

Comment:
Two perspectives below would also profit from same addition.

Reply:
I need to create the icons first. This will take some more time.

Reply:
The different perspectives have different icons and so should the onboarding info.

Reply:
> Two perspectives below would also profit from same addition.

This is done now.